packages/vitest/src/utils/coverage.ts

```diff
@@ -1,6 +1,7 @@
 import { relative } from 'pathe'
 import mm from 'micromatch'
 import type { CoverageMap } from 'istanbul-lib-coverage'
+import type { Vitest } from '../node/core'
```

Comment:
Importing from `vitest/node` here caused a build error 🤔

Reply:
you can't import from vitest/* inside the vitest package. they files are not built yet

Reply:
That makes sense! I guess this relative import is fine here then?